### ISSUE TITLE
Exception when building documents with MeCab (ja)

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#1610]
Setting below in config.py;

```
html_search_laguage = 'ja'
html_search_options = {'type':'mecab'}
```

```
Exception occurred when "make html";
  File "/Library/Python/2.7/site-packages/sphinx/search/ja.py", line 86, in init_ctypes
    self.ctypes_mecab = self.libmecab.mecab_new2(param)
AttributeError: 'MecabBinder' object has no attribute 'libmecab'
```

Sphinx: 1.2.3
MeCab: 0.996
MeCab-Python: 0.996

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/1610
- Originally reported by: Anonymous
- Originally created at: 2014-10-28T12:42:57.231

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2014-10-28 16:10:07+00:00_

Fix #1610: Sphinx cause AttributeError when MeCab search option is enabled and python-mecab is not installed.

→ <<cset 4dd1c538d5105f3f4af60a50233ca851821846f7>>

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2014-10-28 16:15:12+00:00_

If python can do `import MeCab`, this error is not raised. 

When sphinx failed to `import MeCab`, sphinx try to load `libmecab.so` directly by using ctypes. However this situation dig up a potential bug. It's a case of this issue.
